jsapp/js/components/processing/analysis/responseForms/responseWrapper.component.tsx

```diff
@@ -22,6 +22,8 @@ import type { AnalysisQuestionInternal } from '../constants'
 
 interface ResponseWrapperProps {
   uuid: string
+  /** Adds a clear button with the given logic */
+  clearable?: Function
```

Comment:
don't type as `Function` because you always can better be more specific, e.g. `() => unknown`

Reply:
Made the change. Any reason we say `unknown` instead of `void`? If we want to get specific, wouldn't `void` imply that the responses shouldn't return anything and make the parameter clearer?